lib/elastic_record/model.rb

```diff
@@ -18,22 +18,14 @@ def inherited(child)
 
         if child < child.base_class
           child.elastic_index = elastic_index.dup
-          child.doctype = doctype.dup
+          child.elastic_index.model = child
```

Comment:
@matthuhiggins I think this is correct?  This seems to be consistent with how we use inherited models:  children get a separate index.

Reply:
hmm depends how you want polymorphism to work. Would be more consistent with active record if they shared the index unless otherwise specified.